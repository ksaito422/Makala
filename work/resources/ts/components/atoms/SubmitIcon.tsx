import React from 'react';
import { Fab } from '@material-ui/core';
import { SystemUpdateAlt } from '@material-ui/icons';

export const SubmitIcon: React.FC = () => {
  return (
    <Fab color='primary' type='submit'>
      <SystemUpdateAlt />
    </Fab>
  );
}