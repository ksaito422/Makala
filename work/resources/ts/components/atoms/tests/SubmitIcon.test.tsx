import React from 'react';
import renderer from 'react-test-renderer';
import { SubmitIcon } from '../SubmitIcon';

describe('SubmitIcon', () => {
  it('スナップショットテスト', () => {
    const tree = renderer
      .create(<SubmitIcon />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});