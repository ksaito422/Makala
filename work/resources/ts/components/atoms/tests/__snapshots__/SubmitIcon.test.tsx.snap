// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubmitIcon スナップショットテスト 1`] = `
<button
  className="MuiButtonBase-root MuiFab-root MuiFab-primary"
  disabled={false}
  onBlur={[Function]}
  onDragLeave={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex={0}
  type="submit"
>
  <span
    className="MuiFab-label"
  >
    <svg
      aria-hidden={true}
      className="MuiSvgIcon-root"
      focusable="false"
      viewBox="0 0 24 24"
    >
      <path
        d="M12 16.5l4-4h-3v-9h-2v9H8l4 4zm9-13h-6v1.99h6v14.03H3V5.49h6V3.5H3c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h18c1.1 0 2-.9 2-2v-14c0-1.1-.9-2-2-2z"
      />
    </svg>
  </span>
</button>
`;
